src/create_helpers.ts

```diff
@@ -67,83 +67,108 @@ function validateModuleChildren(children?: ts.Node[])
-    if ('returns' in doclet) {
-        return (doclet['returns'] as IDocletReturn[]).map(ret => {
-            if (ret.description) {
+function handleReturnsComment(doclet: IDocletBase): string
+{
+    if ('returns' in doclet)
+    {
+        return (doclet['returns'] as IDocletReturn[]).map((ret) =>
+        {
+            if (ret.description)
+            {
                 return `\n * @returns ${formatMultilineComment(ret.description)}`;
             }
             return '';
-        }).filter(value => value !== '');
+        }).filter((value) => value !== '').join('');
     }
-    return []
+    return ''
 }
 
-function handleExamplesComment(doclet: IDocletBase): string[] {
-    if (doclet.examples !== undefined) {
-        return doclet.examples.map(example => {
+function handleExamplesComment(doclet: IDocletBase): string
+{
+    if (doclet.examples !== undefined)
+    {
+        return doclet.examples.map((example) =>
+        {
             return `\n * @example
  * ${formatMultilineComment(example)}`;
-        });
+        }).join('');
     }
-    return [];
+    return '';
 }
 
-function handleParamsComment(doclet: IDocletBase): string[] {
-    if ('params' in doclet) {
+function handleParamsComment(doclet: IDocletBase): string
+{
+    if ('params' in doclet)
+    {
         return handlePropsComment((doclet['params'] as IDocletProp[]), 'param');
     }
-    return []
+    return ''
 }
 
-function handlePropertiesComment(doclet: IDocletBase): string[] {
-    if (doclet.properties && (!('isEnum' in doclet) || (doclet['isEnum'] === false))) {
+function handlePropertiesComment(doclet: IDocletBase): string
+{
+    if (doclet.properties && (!('isEnum' in doclet) || (doclet['isEnum'] === false)))
+    {
         return handlePropsComment(doclet.properties, 'property');
     }
-    return []
+    return ''
 }
 
 function handleComment<T extends ts.Node>(doclet: IDocletBase, node: T): T
 {
     if (doclet.comment && doclet.comment.length > 4)
     {
         let description = '';
-        if (doclet.description) {
+        if (doclet.description)
+        {
             description = `\n * ${formatMultilineComment(doclet.description)}`;
-        } else if ('classdesc' in doclet) {
+        }
+        else if ('classdesc' in doclet)
+        {
             description = `\n * ${formatMultilineComment((doclet['classdesc'] as string))}`;
         }
         const examples = handleExamplesComment(doclet);
         const properties = handlePropertiesComment(doclet);
         const params = handleParamsComment(doclet);
         const returns = handleReturnsComment(doclet);
 
-        if (description || examples.length > 0 || properties.length > 0 || params.length > 0 || returns.length > 0) {
-            let comment = `*${description}${examples.join('')}${properties.join('')}${params.join('')}${returns.join('')}
+        if (description || examples.length > 0 || properties.length > 0 || params.length > 0 || returns.length > 0)
```

Comment:
The `.length > 0` part can be removed now that it is a string since empty strings are falsey.

```
if (description || examples || properties || params || returns)
```